client/src/plugins/camunda-plugin/deployment-tool/validation/DeploymentConfigValidator.js

```diff
@@ -194,13 +205,22 @@ export default class DeploymentConfigValidator {
     });
   }
 
+  /**
+   * Check if configuration is valid.
+   *
+   * @param {*} configuration
+   * @returns {boolean}
+   */
   isConfigurationValid(configuration) {
 
-    const errors = this.validateBasic(configuration);
+    try {
+      const errors = this.validateBasic(configuration);
 
-    return !hasKeys(errors);
+      return !hasKeys(errors);
+    } catch {
+      return false;
+    }
```

Comment:
Can you explain why we need this try-catch block? Are we expecting the validation crashes in some cases?

Reply:
I'll rewrite this part.